alf/algorithms/algorithm_test.py

```diff
@@ -151,11 +163,14 @@ def test_update_with_gradient(self):
         alg_root.add_optimizer(torch.optim.Adam(lr=0.5), [alg_2])
         loss_info, params = alg_root.update_with_gradient(
             alg_root.calc_loss(TrainingInfo()))
-        self.assertEqual(set(params), set(alg_root.parameters()))
+        self.assertEqual(set(params), set(alg_root.named_parameters()))
         for param in alg_root.parameters():
             self.assertTrue(torch.all(param.grad == 1.0))
         self.assertEqual(loss_info.loss, 3.)
 
 
 if __name__ == '__main__':
+    logging.use_absl_handler()
+    logging.set_verbosity(logging.INFO)
+    AlgorithmTest().test_get_optimizer_info()
```

Comment:
Why ```test_get_optimizer_info()``` needs to be called separately here?

Reply:
fixed